src/tools/crawler/metrics.rs

```diff
@@ -10,7 +10,7 @@ use crate::{
 };
 
 const MIN_BLOCK_HEIGHT: i32 = 2_000_000;
-const ZCASH_P2P_PORT: u16 = 8233;
+pub const ZCASH_P2P_PORT: u16 = 8233;
```

Comment:
the default part implies (for more context) it's not necessarily always the 8233
```suggestion
pub const ZCASH_P2P_DEFAULT_PORT: u16 = 8233;
```

Reply:
Good idea! Changed.